integration-tests.sh

```diff
@@ -65,47 +65,102 @@ echo ""
 echo "####RUNNING TESTS"
 echo ""
 echo "# Running simple estimation of examples/pod.yaml"
-cc --podspec=examples/pod.yaml | tee estimation.log
+$CC --podspec=examples/pod.yaml | tee estimation.log
 if [ -z "$(cat estimation.log | grep 'Termination reason')" ]; then
   printError "Missing termination reason"
   exit 1
 fi
 
 echo ""
 echo "# Running simple estimation of examples/pod.yaml in verbose mode"
-cc --podspec=examples/pod.yaml --verbose | tee estimation.log
+$CC --podspec=examples/pod.yaml --verbose | tee estimation.log
 if [ -z "$(cat estimation.log | grep 'Termination reason')" ]; then
   printError "Missing termination reason"
   exit 1
 fi
 
 echo ""
 echo "# Decrease resource in the cluster by running new pods"
-kubectl create -f examples/rc.yml
+$KUBECTL create -f examples/rc.yml
 if [ "$?" -ne 0 ]; then
   printError "Unable to create additional resources"
   exit 1
 fi
 
-while [ $(kubectl get pods | grep nginx | grep "Running" | wc -l) -ne 3 ]; do
+while [ $($KUBECTL get pods | grep nginx | grep "Running" | wc -l) -ne 3 ]; do
   echo "waiting for pods to come up"
   sleep 1s
 done
 
 echo ""
 echo "# Running simple estimation of examples/pod.yaml in verbose mode with less resources"
-cc --podspec=examples/pod.yaml --verbose | tee estimation.log
+$CC --podspec=examples/pod.yaml --verbose | tee estimation.log
 if [ -z "$(cat estimation.log | grep 'Termination reason')" ]; then
   printError "Missing termination reason"
   exit 1
 fi
 
 echo ""
 echo "# Delete resource in the cluster by deleting rc"
-kubectl delete -f examples/rc.yml
+$KUBECTL delete -f examples/rc.yml
+
+
+echo ""
+echo ""
+echo "# Running with ResourceQuota admission plugin"
+echo "## Creating resource quota"
+$KUBECTL create resourcequota pod-quota --namespace=cluster-capacity --hard=pods=1
+
+$CC --podspec=examples/pod.yaml --resource-space-mode=ResourceSpacePartial --admission-control="ResourceQuota" | tee estimation.log
+if [ -z "$(cat estimation.log | grep 'Admission controller error')" ]; then
```

Comment:
It depends on how many pods were scheduled (either zero or non-zero). This is the message, when I create a pod in cluster-capacity namespace before test run
```
# Running with ResourceQuota admission plugin
## Creating resource quota
resourcequota "pod-quota" created
Unable to create next pod to schedule: Admission controller error: pods "small-pod-0" is forbidden: exceeded quota: pod-quota, requested: pods=1, used: pods=1, limited: pods=1
```

Should we unify output message for both situations or expect them in test?:
 
```sh
...
if [ -z "$(cat estimation.log | grep 'Admission controller error\|AdmissionControllerError')"  ]; then
...
```